pkgs/dart_tooling_mcp_server/lib/src/mixins/dtd.dart

```diff
@@ -541,7 +550,17 @@ base mixin DartToolingDaemonSupport on LoggingSupport, ToolsSupport {
         'This is to apply the latest code changes to the running application. '
         'Requires "${connectTool.name}" to be successfully called first.',
     annotations: ToolAnnotations(title: 'Hot reload', destructiveHint: true),
-    inputSchema: Schema.object(),
+    inputSchema: Schema.object(
+      properties: {
+        'clearRuntimeErrors': Schema.bool(
```

Comment:
can we use a shared const for this string since it is used in multiple tools?

Reply:
Yeah, I will just add it to the bag of constants I created in the other PR, once that lands